src/Model/Response.php

```diff
@@ -78,6 +78,14 @@ public function getNumberOfSkippedCommands(): int
         return $this->numberOfSkippedCommands;
     }
 
+    /**
+     * Return individual command response by its index (0 indexed)
+     */
+    public function getCommandResponse(int $index): CommandResponse
```

Comment:
Wouldn't be better implementing https://github.com/lmc-eu/matej-client-php/issues/8 ?

Reply:
Depends. I think a direct accessor function is desirable.

As for the object storage; I'm inclined to implement `Countable`, `Iterator` and `ArrayAccess` directly, so that you can:

```php
$response[2]->getData();
```

In that case, I'd still prefer a function to get the data as well, as it feels less magical :)